pkg/kudoctl/cmd/package_list.go

```diff
@@ -0,0 +1,57 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+
+	"github.com/kudobuilder/kudo/pkg/kudoctl/clog"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/env"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/packages"
+	pkgresolver "github.com/kudobuilder/kudo/pkg/kudoctl/packages/resolver"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/util/repo"
+)
+
+const packageListDesc = `
+This command consists of multiple sub-commands to interact with KUDO packages.  These commands are used in the listing 
+of an operator details such as parameters, tasks or plans.
+`
+
+const packageListExamples = `  kubectl kudo package list parameters [operator folder]
```

Comment:
because this set of examples is at the `list` level and the examples with great detail are at the lower level (`list plans` or `list params`).   I'll update here do be the same but I don't find this to be the place a user learns anything other than what are all the commands I can use.